data/fixtures/scopes/textual/boundedParagraph.iteration.scope

```diff
@@ -0,0 +1,22 @@
+{
+    foo
+    bar
+}
+---
+
+[#1 Range] =
+[#1 Domain] = 0:0-3:1
+  >-
+0| {
+1|     foo
+2|     bar
+3| }
+   -<
+
+
+[#2 Range] =
+[#2 Domain] = 1:4-2:7
+      >---
+1|     foo
+2|     bar
+   -------<
```

Comment:
would be tempted for domain to extend all the way up to surrounding pairs? At the very least should extend to start of line